tendrl/commons/tests/objects/alert/test_alert_init.py

```diff
@@ -0,0 +1,128 @@
+import __builtin__
+import maps
+from mock import patch
+from tendrl.commons import objects
```

Comment:
Try keeping tendrl imports separate from library imports. We can have a line in between them.